src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -55,31 +56,43 @@ Copyright (c) .NET Foundation. All rights reserved.
     <NETSdkError Condition="'@(_JitResolvedPath)'== ''"
                  ResourceName="UnableToFindResolvedPath"
                  FormatArguments="jit" />
-    
-    
-    
+
     <!-- Get the crossgen and jit path-->
     <PropertyGroup>
       <_CoreclrPath>@(_CoreclrResolvedPath)</_CoreclrPath>
-      <_JitPath>@(_JitResolvedPath)</_JitPath>
+      <JitPath>@(_JitResolvedPath)</JitPath>
       <_CoreclrDir>$([System.IO.Path]::GetDirectoryName($(_CoreclrPath)))</_CoreclrDir>
       <_CoreclrPkgDir>$([System.IO.Path]::Combine($(_CoreclrDir),"..\..\..\"))</_CoreclrPkgDir>
-      <_CrossgenDir>$([System.IO.Path]::Combine($(_CoreclrPkgDir),"tools"))</_CrossgenDir>
+      <CrossgenDir>$([System.IO.Path]::Combine($(_CoreclrPkgDir),"tools"))</CrossgenDir>
       <!-- TODO override with rid specific tools path for x-arch -->
-      <_Crossgen>$([System.IO.Path]::Combine($(_CrossgenDir),"crossgen"))</_Crossgen>
-      <_Crossgen Condition="$(OS.ToUpper().Contains('WINDOWS'))">$([System.IO.Path]::Combine($(_CrossgenDir),"crossgen.exe"))</_Crossgen>
+      <Crossgen>$([System.IO.Path]::Combine($(CrossgenDir),"crossgen"))</Crossgen>
+      <Crossgen Condition="$(OS.ToUpper().Contains('WINDOWS'))">$([System.IO.Path]::Combine($(CrossgenDir),"crossgen.exe"))</Crossgen>
```

Comment:
`$(OS.ToUpper().Contains('WINDOWS'))`

1. MSBuild does a case-insensitive comparison by default.
2. The way we do this check everywhere else in the repo is `'$(OS)' == 'Windows_NT'`